x-pack/plugins/lens/public/drag_drop/drag_drop.tsx

```diff
@@ -248,3 +305,207 @@ const DragDropInner = React.memo(function DragDropInner(
     onDragStart: dragStart,
   });
 });
+
+const getKeyboardReorderMessageMoved = (position: number, prevPosition: number) =>
+  i18n.translate('xpack.lens.dragDrop.elementMoved', {
+    defaultMessage: 'You have moved the item from position {prevPosition} to position {position}',
+    values: {
+      position,
+      prevPosition,
+    },
+  });
+
+const getKeyboardReorderMessageLifted = (position: number) =>
+  i18n.translate('xpack.lens.dragDrop.elementLifted', {
+    defaultMessage: 'You have lifted an item in position {position}',
```

Comment:
I found this screenreader message hard to understand because it wasn't telling me the item name again

Reply:
I agree, that was the initial version but I then changed it to be consistent with EUI copy. Changing it back :)